### DISCUSSION TITLE
公開アバターを利用する際のスケール調整について

### DISCUSSION THREAD

**Discussion by Rooooki129** [pixiv/vroid-sdk-developers#112] [Q&A]
VRoid SDKを用いてVRoid Hubに公開されているアバターをインポートし利用する際、アプリ内のワールドのスケールに合わせてインポートしたアバターのスケールを変更することは「改変」には該当しない認識ですが、これは正しいでしょうか。

具体的には、アプリで規定する高さの範囲外のアバターについてはアプリ側でスケール調整し利用したいと考えております。

**Comment by nyamadan** [comment]
改変については[VRMの仕様](https://vrm.dev/licenses/1.0/)に従っていただく必要がございます。
たとえば第2条a7に以下のような記述があります。

>「ライセンサー」は、「あなた」に対し、あらゆる媒体や形式（現在知られているか又は今後考案されるかを問いません）で「ライセンスされた権利」を行使することのできる権限を付与し、当該行使に必要とされる技術的な改変を行う権限を付与します。

お手数おかけしますが、ライセンス文章を参考のうえ、今回の @Rooooki129  さまのケースに該当するかどうか吟味いただけますと幸いです。
